client/components/domains/connect-domain-step/index.jsx

```diff
@@ -61,11 +61,16 @@ function ConnectDomainStep( {
 	const [ loadingDomainSetupInfo, setLoadingDomainSetupInfo ] = useState( false );
 
 	const baseClassName = 'connect-domain-step';
-	const isStepStart = stepType.START === stepsDefinition[ pageSlug ].step;
-	const mode = stepsDefinition[ pageSlug ].mode;
-	const step = stepsDefinition[ pageSlug ].step;
-	const prevPageSlug = stepsDefinition[ pageSlug ].prev;
-	const isTwoColumnLayout = ! stepsDefinition[ pageSlug ].singleColumnLayout;
+	if ( stepsDefinition[ pageSlug ] === undefined ) {
+		// eslint-disable-next-line no-console
+		console.error( 'Tried to set invalid pageSlug in ConnectDomainStep', pageSlug, firstStep );
```

Comment:
Do these errors appear in production? I think warnings are hidden, but I don't remember if errors are too.

Reply:
I think those will even get logged in the calypso elassticsearch index. So we might get some details on what's actually happening and figure out how to fix it.

Reply:
@hambai do you think it might help to include the domain name in the error?